src/redux/books/books.js

```diff
@@ -23,5 +27,27 @@ export const removeBook = (payload) => ({
   type: REMOVE_BOOK,
   payload,
 });
+export const addBookToAApi = ({ itemId, title, category }) => function middle(dispatch) {
+  axios.post(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${apiKey}/books`, {
+
+    item_id: itemId,
+    title,
+    category,
+
+  })
+    .then((response) => {
+      if (response.status === 201) {
+        dispatch(addBook({ itemId, title, category }));
+      }
+    });
+};
+
+export const removeBookFromApi = (id) => function middle(dispatch) {
+  axios.delete(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${apiKey}/books/${id}`,
+    { item_id: id })
+    .then(() => {
+      dispatch(removeBook(id));
+    });
+};
```

Comment:
[OPTIONAL] I think we left out an important feature during this milestone. Is that when the page loads it retrieves the books stored already on the API and displays them. This way if the user reloads the page or goes to a different computer is still has access to the books on the API.

I noticed this is not specified on the requirements so I will mark it as optional. If you implement this, make sure that if you go to categories, then come back to the booklist the books are not fetched twice.